monai/engines/evaluator.py

```diff
@@ -146,7 +147,28 @@ def run(self, global_epoch: int = 1) -> None:
         self.state.iteration = 0
         super().run()
 
-    def get_validation_stats(self) -> dict[str, float]:
+    def get_stats(self, *vars):
+        """
+        Get the statistics information of the validation process.
+        Default to return the `rank`, `best_validation_epoch` and `best_validation_metric`.
+
+        Args:
+            vars: except for the default stats, other variables name in the `self.state` to return,
+                will use the variable name as the key and the state content as the value.
+                if the variable doesn't exist, default value is `None`.
+
+        """
+        stats = {
+            ESKeys.RANK: self.state.rank,
+            ESKeys.BEST_VALIDATION_EPOCH: self.state.best_metric_epoch,
+            ESKeys.BEST_VALIDATION_METRTC: self.state.best_metric,
+        }
+        for k in vars:
+            stats[k] = getattr(self.state, k, None)
+        return stats
+
+    @deprecated(since="0.8", msg_suffix="please use the `get_stats()` API instead.")
```

Comment:
this should be since 0.9?

Reply:
Sure, updated in the latest commit.

Thanks.